### ISSUE TITLE
space between two `\circ`

### ISSUE THREAD

**Issue by hbghlyj** [ronkok/Temml#43]
Example:
```tex
\begin{array}{c}
1{\circ}{\circ}1\\
1{\circ\circ}1
\end{array}
```
<table>
<tr><td>

In Temml, the spacing between $\circ\circ$ are different in the two lines
</td><td>

![image](https://github.com/ronkok/Temml/assets/53823634/5f5e1082-3fa6-46c0-8952-75c76591690f)

</td></tr>
<tr><td>

In [LaTeX](https://texlive.net/run?%5Cdocumentclass%7Barticle%7D%0A%5Cbegin%7Bdocument%7D%0A%24%5Cbegin%7Barray%7D%7Bc%7D%0A1%7B%5Ccirc%7D%7B%5Ccirc%7D1%5C%5C%0A1%7B%5Ccirc%5Ccirc%7D1%0A%5Cend%7Barray%7D%24%0A%5Cend%7Bdocument%7D),
the two lines are the same:
</td>
<td><p align="center">
<img src=https://github.com/ronkok/Temml/assets/53823634/b347b69f-ad5e-40d9-9648-ed1b3386df03>
</p></td></tr>
</table>

**Comment by ronkok** [comment]
Hmm... interesting. This is related to issue #31. I'll look into it. No promises.

**Comment by ronkok** [comment]
Fixed by release v0.10.20.
